### PR TITLE
Chore: Update actions

### PR DESCRIPTION
**Pull request by bvotteler**
~~Remove author field in `package.json` .~~ (undone; to be confirmed if we want to use `contributors` instead of `author` field)
Update github actions to use `setup-node@v3` with `node-version-file` to install node version specified in `.nvmrc`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
   "description": "polkawallet bridge sdk",
   "main": "build/src/index.js",
   "typings": "build/src/index.d.ts",
-  "author": "Polkawallet developers <hello@polkawallet.io>",
```
Comment (tomjeatt) [inline package.json:7]: @dom had some guidance for this:
https://www.notion.so/Licensing-and-Forking-Code-2f74bea9c577458cb449a9ca59c815ec

**Review by tomjeatt — CHANGES_REQUESTED** [review]
Looks good Brendon :) one change around the author field, as Dom shared some guidance for licensing and forking open source code.

File: package.json (hunk 1 above)
Comment (bvotteler) [reply package.json]: > @dom had some guidance for this
Good shout, thanks.
We cannot add more people to the `author` field, as it only holds a singular entity.

I'll undo this for now, to be clarified if we want to change to using a `contributors` field instead.

Comment (bvotteler) [reply package.json]: With regards to the author vs contributors field, see also this [npm doc](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#people-fields-author-contributors)

**Review by tomjeatt — APPROVED** [review]
💥

File: package.json (hunk 1 above)
Comment (nud3l) [reply package.json]: I think it best to replace `authors` with `contributors` and use an array with the Polkawallet developers and us in there.

**Pull request merged**